xbmc/cores/VideoRenderers/BaseRenderer.cpp

```diff
@@ -222,11 +222,14 @@ void CBaseRenderer::FindResolutionFromFpsMatch(float fps, float& weight)
 RESOLUTION CBaseRenderer::FindClosestResolution(float fps, float multiplier, RESOLUTION current, float& weight)
 {
   RESOLUTION_INFO curr = g_graphicsContext.GetResInfo(current);
-
+  RESOLUTION_INFO orig = curr;
```

Comment:
the method it gets called with - it was there before me - i implicitely thought it is RES_DESKTOP